### PR TITLE
[docs] Shap-E guide

### PR DESCRIPTION
**Pull request by stevhliu**
Creates a standalone Shap-E guide to reduce bloat on the API page and also fixes some typos in the docstrings.

### CODE REVIEW

File: docs/source/en/api/utilities.md [hunk 1]
```diff
@@ -18,6 +18,10 @@ Utility and helper functions for working with 🤗 Diffusers.
 
 [[autodoc]] utils.testing_utils.load_image
 
+## export_to_gif
```
Comment (yiyixuxu) [inline docs/source/en/api/utilities.md:21]: awesome:)

File: docs/source/en/using-diffusers/shap-e.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
+# Shap-E
+
+[[open-in-colab]]
+
+Shap-E is a conditional model for generating 3D assets which could be used for video game development, interior design, and architecture. It is trained on a large dataset of 3D assets, and post-processed to render more views of each object and produce 16K instead of 4K point clouds. The Shap-E model is trained in two steps:
+
+1. a encoder accepts the point clouds and rendered views of a 3D assets and outputs the latents of that 3D asset which are used to produce the asset's weights
```
Comment (yiyixuxu) [inline docs/source/en/using-diffusers/shap-e.md:7]:
```suggestion
1. a encoder accepts the point clouds and rendered views of a 3D assets and outputs the parameters of implicit functions that represent the assetd
```

**Review by yiyixuxu — APPROVED** [review]
thanks!

**Review by patrickvonplaten — APPROVED** [review]
Thanks!

**Pull request merged**